### DISCUSSION TITLE
Possibility to access all OBD data through the app

### DISCUSSION THREAD

**Discussion by AlexBimmerHead** [BimmerGestalt/AAIdrive#753] [Ideas]
What data are accessible through the current version of AAIdrive? It is possible to add other engine data like, for example, boost pressure?

**Comment by hufman** [comment]
The only datapoints that are available are documented here: https://bimmergestalt.github.io/BMWConnectedAnalysis/cds/

**Reply by AlexBimmerHead** [reply]
Thank you!!
